examples/viewer.py

```diff
@@ -282,6 +282,37 @@ def key_press_event(self, event: Application.KeyEvent) -> None:
         elif key == pressed.H:
             self.print_help_text()
 
+        elif key == pressed.TAB:
+            # NOTE: (+ALT) - reconfigure without cycling scenes
+            if not alt_pressed:
+                # cycle the active scene from the set available in MetadataMediator
+                inc = -1 if shift_pressed else 1
+                scene_ids = self.sim.metadata_mediator.get_scene_handles()
+                cur_scene_index = 0
+                if self.sim_settings["scene"] not in scene_ids:
+                    matching_scenes = [
+                        (ix, x)
+                        for ix, x in enumerate(scene_ids)
+                        if self.sim_settings["scene"] in x
```

Comment:
I could foresee a problem here if the specified scene in sim_settings (which seems to be reading the value initially from args) is a non-unique substring of the scene lists (i.e. "apt" was specified in replicaCAD, for example).  This would stop paging from occurring.

Ideally self.sim_settings["scene"] should be set to the name of the currently loaded scene instance after every call to reconfigure, to avoid this.   This is what is happening in viewer.cpp already.  I am fixing this in viewer.py in [this PR](https://github.com/facebookresearch/habitat-sim/pull/1632).